index-de.html

```diff
@@ -76,7 +76,7 @@ <h2>Die längere Fassung</h2>
       Solltest du dich belästigt fühlen oder mitbekommen, dass jemand anders
       belästigt wird oder sonstige Probleme auftreten, wende dich bitte sofort
       an ein Mitglied des Veranstaltungspersonals. Diese sind an ihren
-      entsprechend markierten T-Shirts schnell zu erkennen.
+      entsprechend markierten T-Shirts und/oder Ausweisen schnell zu erkennen.
```

Comment:
This adds the badges as marker, which is a good thing. Not every organizer uses special t-shirts.

Reply:
The badges are also mentioned in the original, but were missing in the German translation!

Reply:
Ah, I see. Need more coffee.